src/styles/blocks/main-navigation.scss

```diff
@@ -0,0 +1,75 @@
+@import "../utils/variables";
+
+.main-navigation {
+  margin-right: 50px;
+  overflow-x: scroll;
+  overflow-y: hidden;
+
+  &__list {
+    height: 60px;
+    display: flex;
+    margin: 0;
+    line-height: 60px;
+    padding: 0;
+  }
+  &__item {
+    list-style: none;
+    text-transform: uppercase;
+    height: 60px;
+    display: flex;
+    flex-direction: column;
+    color: $main-accent;
+  }
+
+  &__item:not(:first-of-type) {
+    margin-left: 20px;
+  }
+
+  &__item:first-of-type::after {
+    position: relative;
+    bottom: 4px;
+    content: "";
+    background-color: #00acdc;
+    height: 0;
+    padding: 2px 0;
+    border-radius: 2px;
+    width: 100%;
+  }
+
+  &__item > a {
+    text-decoration: none;
+    color: $main-accent;
+    transition-duration: 0.3s;
+  }
+
+  &__item > a:hover {
+    color: $blue-accent;
+  }
+  .is-active {
+    color: $blue-accent;
+  }
+}
+
+.logo {
+  align-self: center;
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.main-page {
+  min-width: 944px;
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.container {
```

Comment:
You have another file for `.container` styling. Put all related styles to one place.

Reply:
you moved those styles to separate file but forget to delete here 😉